app/views/results/edit.html.erb

```diff
@@ -15,6 +16,7 @@
           grouping_id: <%= @result.submission.grouping_id %>,
           assignment_id: <%= @result.submission.grouping.assessment_id %>,
           role: '<%= @current_user.class.name %>',
+          num_collected: <%= @assignment.get_num_collected(ta_id) %>,
```

Comment:
this shouldn't be passed as a prop. Instead, get the information when setting the `state` of the Result component.